networkd_db/networkd_db.ml

```diff
@@ -43,7 +43,10 @@ let _ =
 			if List.mem_assoc !bridge config.bridge_config then begin
 				let bridge_config = List.assoc !bridge config.bridge_config in
 				let ifaces = List.flatten (List.map (fun (_, port) -> port.interfaces) bridge_config.ports) in
```

Comment:
If I'm not mistaken, below `ifaces` is only used as a string - so the conversion to a string could happen here once:

```
let ifaces = 
  bridge_config.ports
  |> List.map (fun (_, port) -> port.interfaces
  |> List.flatten
  |> String.concat ","
in
```